docs/release/2023q1.md

```diff
@@ -0,0 +1,230 @@
+## 2023-04-06: 2023q1 release
+
+This release covers everything from 2023-01-01 and has been tested with avr-gcc
+v12.2.0 from Upstream and arm-none-eabi-gcc 12.2.rel1 from Arm.
+
+
+Breaking changes:
+
+- SAMV71 Xplained Ultra button polarity.
+
+Features:
+
+- External main clock on SAMx7x.
+- SAMx7x SPI driver.
+- SAM GpioInverted.
+- CANFD large frame support.
+- SAM SoftwareGpioPort.
+- SAMx7x Timer/counter driver.
+- SAMx7x PWM generator driver.
+- GCC 12.
+- Ability to coredump from GDB directly.
+- JLink support.
+- Support for STM32U5 devices.
+- Repetition Counter on STM32 Advanced Control Timers.
+- ADC scan mode, DMA mode and external trigger for STM32F1/F2/F37/F4/F7/L1.
+- SAMx7x I2C driver.
+
+Integrated Projects:
+
+- ETL upgraded to v20.35.14.
+
+Fixes:
+
+- Multiple fixes for I²C NACK handling on STM32.
+- On Nucleo-F103RB GpioB4 is now usable.
+- Systick frequency on SAMx7x.
+
+New development boards:
+
+- Nucleo-U575ZI-Q as `modm:nucleo-u575zi-q`
```

Comment:
```suggestion
- Nucleo-U575ZI-Q as `modm:nucleo-u575zi-q`.
```
Otherwise it won't get auto-linked by the [sync docs script](https://github.com/modm-io/modm/blob/develop/tools/scripts/synchronize_docs.py#L195).
Same with the drivers. (Alternatively, the script regex could be fixed)